test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo14-addon-knowledge @git+https://github.com/dzungtran89/knowledge@14.0-mig-knowledge#subdirectory=setup/knowledge
```

Comment:
please drop these

Reply:
Thanks @simahawk. This PR has been updated.